src/SFA.DAS.Reservations.Web.UnitTests/Models/WhenConstructingAReservationViewModel.cs

```diff
@@ -59,11 +63,12 @@ public class WhenConstructingAReservationViewModel
         public void And_Course_Is_Null_Then_Sets_CourseDescription_To_Unknown(
             Reservation reservation,
             string url,
-            string alephid)
+            string accountHashedId,
+            string accountLegalEntityPublicHashedId)
```

Comment:
In this file a number of accountHashedIds and accountLegalEntityPublicHashedId do not seem to be used but are in the parameters. Can they be removed?